.gitignore

```diff
@@ -4,3 +4,4 @@ moduleIndex.json
 last-moduleIndex.json
 generated-moduleIndex.json
 .DS_Store
+.vscode/settings.json
```

Comment:
Should be removed. We're setting this explicitely to align the settings for the workspace (i.e., avoid different ident levels, formatters etc. that create a lot of noise)

Reply:
done